phing/tasks/blt.xml

```diff
@@ -73,6 +73,10 @@
     <exec dir="${repo.root}" command="${blt.root}/scripts/blt/remove-deprecated.sh" logoutput="true" checkreturn="true" level="${blt.exec_level}" passthru="true" />
   </target>
 
+  <target name="version" description="Display the currently installed version of BLT.">
+      <exec dir="${repo.root}" command="echo 'BLT version:' $(composer show | grep blt | awk '{print $2}')" logoutput="true" checkreturn="true"/>
```

Comment:
I'd also add a `level="${blt.exec_level}"` so that it conforms to the BLT verbosity settings.